docs/home/contribute/includes.md

```diff
@@ -1,12 +1,66 @@
 ---
 approvers:
 - chenopis
-title: Tabs Example
+title: Standard Include Statements
```

Comment:
Who is our audience? If we can assume that they are familiar with Jekyll (or that we only want people with that background reading this), this is fine. Otherwise, it might be a bit unclear what this section pertains to (and a new contributor skimming the sidebar might miss it).

I suggest something along the lines of *Leveraging custom UI elements with includes statements* or a shorter version of that. Let me know if you disagree.

Reply:
Changed the title to "Custom Jekyll Include Snippets." Let me know if that works!